source/crud/read-operations/skip.txt

```diff
@@ -0,0 +1,58 @@
+=====================
+Skip Returned Results
+=====================
+
+.. default-domain:: mongodb
+
+
+Use skip to omit documents from the beginning of the list of returned
+documents for a read operation. You can combine skip with sort to omit
+the top (for descending order) or bottom (for ascending order) ``N``
```

Comment:
Users might not be familiar with using "N" to represent an integer.

Reply:
The sentence is pretty clear even without N, so I'm removing this entirely. Thanks!